packer/studio-selfhosted.pkr.hcl

```
@@ -0,0 +1,129 @@
+packer {
+  required_plugins {
+    amazon = {
+      version = ">= 1.0.0"
+      source  = "github.com/hashicorp/amazon"
+    }
+  }
+}
+
+variables {
+  image_name             = "studio-selfhosted"
+  image_description      = "Iterative Studio Selfhosted - {{isotime `2006-01-02`}}"
+  aws_build_region       = "us-west-1"
+  aws_build_instance     = "m6i.large"
+  aws_build_ubuntu_image = "*ubuntu-*-22.04-amd64-server-*"
+  skip_create_ami        = true
+}
+
+locals {
+  aws_tags = {
+    ManagedBy   = "packer"
+    Name        = var.image_name
+    Environment = "prod"
+    BuildDate   = "{{isotime `2006-01-02`}}"
+  }
+
+  aws_release_regions = [
+    #    "af-south-1",
+    #    "ap-east-1",
+    "ap-northeast-1",
+    "ap-northeast-2",
+    "ap-northeast-3",
+    #    "ap-south-1",
+    #    "ap-south-2",
+    #    "ap-southeast-1",
+    #    "ap-southeast-2",
+    #    "ap-southeast-3",
+    #    "ap-southeast-4",
+    "ca-central-1",
+    #    "eu-central-1",
+    #    "eu-central-2",
+    "eu-north-1",
+    #    "eu-south-1",
+    #    "eu-south-2",
+    "eu-west-1",
+    "eu-west-2",
+    "eu-west-3",
+    #    "me-central-1",
+    #    "me-south-1",
+    #    "sa-east-1",
+    "us-east-1",
+    "us-east-2",
+    "us-west-1",
+    "us-west-2",
```

```suggestion
    #    "af-south-1",
    #    "ap-east-1",
    "ap-northeast-1",
    "ap-northeast-2",
    "ap-northeast-3",
    "ap-south-1",
    #    "ap-south-2",
    "ap-southeast-1",
    "ap-southeast-2",
    #    "ap-southeast-3",
    #    "ap-southeast-4",
    "ca-central-1",
    "eu-central-1",
    #    "eu-central-2",
    "eu-north-1",
    "eu-south-1",
    #    "eu-south-2",
    "eu-west-1",
    "eu-west-2",
    "eu-west-3",
    #    "me-central-1",
    "me-south-1",
    "sa-east-1",
    "us-east-1",
    "us-east-2",
    "us-west-1",
    "us-west-2",
```

Added the remaining regions. I've verified that we have them in our AWS account

---

I enabled all regions 🎉